Motorcycle.js

```diff
@@ -1,6 +1,34 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(make, model, year, color, seats) {
+	Car.call(this, make, model, year, color, seats);
```

Comment:
We don't need a function to handle the max seats, just a simple override in our constructor using an if statement:
```
if (this.seats > 2) {
		this.seats = 2;
	}
```